packages/user-operation-controller/src/helpers/Bundler.test.ts

```diff
@@ -23,108 +23,96 @@ const USER_OPERATION_MOCK: UserOperation = {
 
 describe('Bundler', () => {
   describe.each([
-    ['estimateUserOperationGas', [USER_OPERATION_MOCK, ENTRYPOINT_MOCK]],
-    ['getUserOperationReceipt', [HASH_MOCK]],
-    ['sendUserOperation', [USER_OPERATION_MOCK, ENTRYPOINT_MOCK]],
-  ])('%s', (methodName, args) => {
-    it('sends JSON-RPC request to URL', async () => {
+    [
+      'estimateUserOperationGas' as const,
+      [USER_OPERATION_MOCK, ENTRYPOINT_MOCK],
+    ],
+    ['getUserOperationReceipt' as const, [USER_OPERATION_HASH_MOCK]],
+    ['sendUserOperation' as const, [USER_OPERATION_MOCK, ENTRYPOINT_MOCK]],
+  ])(
+    '%s',
+
+    (methodName, args) => {
       const bundler = new Bundler(URL_MOCK);
 
-      jest.spyOn(global, 'fetch').mockResolvedValueOnce({
-        json: () =>
-          Promise.resolve({
-            result: {},
+      /**
+       * Invoke the current method on the bundler instance.
+       */
+      async function invokeBundler() {
```

Comment:
```suggestion
      async function invokeBundlerMethod() {
```

Reply:
This feels implied as methods are what we invoke in this context, but I appreciate the importance of naming for promoting readability.